Sources/ConsoleKit/Command/Base/CommandRunnable.swift

```diff
@@ -1,23 +1,64 @@
-/// Capable of being run on a `Console` using `Console.run(...)`.
-/// - note: This base protocol should not be used directly. Conform to `Command` or `CommandGroup` instead.
-public protocol CommandRunnable {
-    /// The supported options.
-    var options: [CommandOption] { get }
-
+/// A type-erased `CommandRunnable`.
+public protocol AnyCommandRunnable {
+    /// An instance of the type that represents the command's valid inputs/signature.
+    static var inputs: CommandSignature { get }
```

Comment:
Would there be any utility in making this an instance variable so that different instantiations of the same command type could have a slightly different signature? That's possible in Console 3.0, but I'm not sure if anyone actually uses it or cares.

Reply:
Maybe we should call this `_signature` or `anySignature` (since it's type-erased).

Reply:
I honestly have never seen this done, and I think there are some cases in the code that would make it very difficult to make that an instance variable. I think we should just leave at this point. If nothing comes up in alpha/beta we'll call it all clear. If anyone mentions they need it, we can look into it then.